src/combinatorics/set_family/compression/uv.lean

```diff
@@ -79,8 +76,9 @@ variables [generalized_boolean_algebra α] [decidable_rel (@disjoint α _ _)]
 
 local attribute [instance] decidable_eq_of_decidable_le
```

Comment:
Out of scope but this looks like trouble to me

Reply:
Agreed, but it's a bit of a mess to fix because `compress` doesn't need `decidable_eq`.